libs/platform/src/lib/thumbnail/thumbnail-image/thumbnail-image.component.html

```diff
@@ -6,7 +6,7 @@
         [class.fdp-thumbnail-image--selected]="image.selected"
         [class.fdp-thumbnail-image--overlay]="image.overlayRequired"
         (click)="image.overlayRequired ? openDialog(image, mediaList) : thumbnailClick(image)"
-        (keydown.space)="image.overlayRequired ? openDialog(image, mediaList) : thumbnailClick(image)"
+        (keydown)="image.overlayRequired ? openDialog(image, mediaList) : thumbnailClick($event, image)"
```

Comment:
why did you remove `space`?
does it mean that any `keydown` will lead to `onClick` event firing?